spec/dummy/app/assets/stylesheets/active_admin.css

```diff
@@ -0,0 +1,17 @@
+// SASS variable overrides must be declared before loading up Active Admin's styles.
```

Comment:
are these valid css comments? look weird in Github and in https://developer.mozilla.org/en-US/docs/Web/CSS/Comments says that comments are like `/* ... */`

Reply:
Will check!